docs/src/main/asciidoc/health-guide.adoc

```diff
@@ -296,6 +296,15 @@ error along with the health check response.
         }
 ----
 
+== Extension health checks
+
+Some extension may provides default health checks, including the extension will automatically register its health checks.
```

Comment:
Change "extension may provides" to "extensions may provide".

Reply:
done